lib/drivers/tandemDriver.js

```diff
@@ -1595,6 +1595,14 @@ module.exports = function (config) {
           debug('Event with unhandled change type:', event);
           throw new Error('Unhandled combination of basal change types: ' + event.change_types.join('|'));
       }
+      if(postbasal.deliveryType === 'temp') {
```

Comment:
You can't have this logic above the `if (postbasal != null)` - errored on my pump.